src/include/simeng/Elf.hh

```diff
@@ -12,13 +12,13 @@ const char Format32 = 1;
 const char Format64 = 2;
 }  // namespace ElfBitFormat
 
-struct ElfHeader {
-  uint32_t type;
-  uint64_t offset;
-  uint64_t virtualAddress;
-  uint64_t physicalAddress;
-  uint64_t fileSize;
-  uint64_t memorySize;
+struct Elf64_Phdr {
```

Comment:
Given the name of the member, variables have been changed to be Linux-specific, it isn't obvious what each one represents. It would be good to add comments explaining what each one represents as well as a small description about `Elf64_Phdr`